instrumentation/rack/lib/opentelemetry/instrumentation/rack/middlewares/tracer_middleware.rb

```diff
@@ -6,8 +6,6 @@
 
 require 'opentelemetry/trace/status'
 
-require_relative '../util/queue_time'
-
 module OpenTelemetry
   module Instrumentation
     module Rack
```

Comment:
Any reason not to update this to use the `config[:allowed_rack_request_headers]` and `config[:allowed_rack_response_headers]` precomputed in the `Instrumentation` class?